CHANGES.rst

```diff
@@ -32,6 +32,7 @@ Changelog
 
 **Changed**
 
+- #455 On the AR View Date Published should appear once
```

Comment:
Please, updated this branch to the latest master and move this log from "1.1.8 (2017-12-23)" to "1.2.1 (unreleased)" section.